rootfs-builder/rootfs.sh

```diff
@@ -383,6 +383,7 @@ fi
 OK "Agent installed"
 
 [ "${AGENT_INIT}" == "yes" ] && setup_agent_init "${AGENT_DEST}" "${init}"
+ln -sf /sbin/init "${ROOTFS_DIR}/init"
```

Comment:
This doesn't look right as you are creating a link to the hosts init inside the rootfs. Shouldn't it be something like the following?

```
from=/lib/systemd/systemd
to=/sbin/init

[ "${AGENT_INIT}" == "yes" ] && from=/usr/bin/kata-agent

sudo chroot "${ROOTFS_DIR} ln -sf "$from" "$to"
```